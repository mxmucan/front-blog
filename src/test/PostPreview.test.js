import React from 'react';
import { shallow } from 'enzyme';
import PostPreview from '../components/PostPreview';

describe('<PostPreview/>', () => {
    it ('Simple Reader Works', () => {
        const component = shallow(<PostPreview/>);
        expect(component).toMatchSnapshot();
    });

    it ('Render with props works', () => {
        const component = shallow(
            <PostPreview _id={'1234567890'} title={'post test'}/>
        );
        expect(component.find('.post-title').text()).toBe('post test');
    });

});