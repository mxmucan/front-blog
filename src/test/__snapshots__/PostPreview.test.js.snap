// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostPreview/> Simple Reader Works 1`] = `
<div
  className="post-preview"
>
  <Link
    to="/post/undefined"
  >
    <h2
      className="post-title"
    />
    <h3
      className="post-subtitle"
    >
      Problems look mighty small from 150 miles up
    </h3>
  </Link>
  <p
    className="post-meta"
  >
    Posted by
    <a
      href="#"
    >
      Start Bootstrap
    </a>
    on September 24, 2019
  </p>
</div>
`;
